MCP_Server/tools/transport_tools.py

```
@@ -0,0 +1,237 @@
+def register(mcp: FastMCP, get_connection, cache):
+    """Register transport tools with the MCP server"""
+
+    @mcp.tool()
+    async def start_playback() -> str:
+        """Start playing the session."""
+        try:
+            conn = await get_connection()
+            result = await conn.send_command("start_playback", {})
+            cache.invalidate_all()
+            return json.dumps(result, indent=2)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)